bin/sl-run.js

```diff
@@ -54,7 +54,7 @@ if ((config.clustered && config.isMaster) || config.detach) {
   return config.start();
 }
 
-if (config.enableTracing) {
+if (require('strongloop-license')('strong-trace:tracer')) {
```

Comment:
This conflates the explicit request to enable tracing (https://github.com/strongloop/strong-supervisor/blob/master/lib/options.js#L104-L106) with whether tracing is licensed. Tracing should not occur unless requested (`enableTracing` is true). If enabled, it should not be started unless it's licensed.

Anyway, the license check should be made in strong-trace (as it is for strong-agent).

Otherwise, strong-agent can be used without a license by just directly npm installing it, and using it.